### ISSUE TITLE
Login and Sign up to contribute should be closed to each other and have the same look

### ISSUE THREAD

**Issue by Natouille** [tosdr/phoenix#224]
As it is similar actions (sign in / sign up), it should be closed to each other (this is a best practice, usualy at the top right of the site, with the same look, so people should know where to find them).

I would recommand : "Sign up to contribute" (button) - "Login (or Signin)" (button) - About (simple link)

**Comment by piks3l** [comment]
Agreed. Do you think that moving the login button that is on the right of the navbar next to the button "Sign in to contribute" will do the trick ? Do they have to have the same layout ? Usually, it's different... I never understood why though.

**Comment by Natouille** [comment]
You could put them on the left, sure. I think it's better if they have the same look. Many sites have a different look for sign in and sign up and some even put a biggest focus on the Signup action (and Signin is lost in the template, usually smaller)
